### ISSUE TITLE
Missing `config` configuration support for model `columns`

### ISSUE THREAD

**Issue by jeremyyeo** [dbt-labs/dbt-jsonschema#213]
Columns now have a `config` key - https://docs.getdbt.com/reference/resource-properties/columns

```yaml
models:
  - name: foo
    columns:
      - name: id
        config:  # <<<<<< Config on a column.
          meta: ...
```

We haven't added this to the json schema yet (https://github.com/dbt-labs/dbt-jsonschema/blob/main/schemas/latest/dbt_yml_files-latest.json#L1189) so this is throwing a false positive warning for linting tools that depend on it - e.g.

### VS Code + YAML Extension

<img width="634" height="271" alt="Image" src="https://github.com/user-attachments/assets/1496849f-3b39-4dd8-9324-596a6e26738c" />

### dbt (Platform) Studio

<img width="678" height="373" alt="Image" src="https://github.com/user-attachments/assets/ed89693c-0e8d-4d56-9bf7-2c87a471dd94" />

**Comment by Intoxinator** [comment]
@jeremyyeo I have an update that seems to work - https://gist.github.com/Intoxinator/ec08c317cc892fa853b5f9a009caac34

If you can spin up a branch for me, I'll put up a PR.

**Comment by eugenekim-orrum** [comment]
Would love an update here!

**Comment by MikeMKH** [comment]
Any update on this yet?

**Comment by tilsch25** [comment]
Hey @jeremyyeo  , are there any plans to implement this? Would you be open to review a PR that implemented this?
